india_compliance/patches/post_install/improve_item_tax_template.py

```diff
@@ -52,6 +52,9 @@
 
 
 def execute():
+    if not frappe.db.has_column("Item", "is_nil_exempt"):
+        return
```

Comment:
update_gst_details_for_transaction can run multiple times.

Reply:
The column is still present in the database even after we delete the custom field. This is to ensure the patch doesn't run for a completely fresh site.